src/components/map/MapComponent.vue

```diff
@@ -43,7 +71,12 @@ layerStore.$subscribe(() => {
   <div class="absolute right-2 top-2 z-10">
     <TransportButtons></TransportButtons>
   </div>
-  <div class="absolute right-2 bottom-2">
-    <UiButton @click="toggleMap">{{ is3D() ? '2D' : '3D' }}</UiButton>
+  <div class="absolute right-4 bottom-2 flex flex-col [&>*]:m-2 text-[#5E5E5E]">
```

Comment:
better not to use custom color here. It's better to put declare it on tailwind.config.cjs